setup.py

```diff
@@ -3,7 +3,7 @@
 """
     Purity//FB REST Client
 
-    Client for Purity//FB REST API (1.0 - 1.8), developed by [Pure Storage, Inc](http://www.purestorage.com/). Documentations can be found at [purity-fb.readthedocs.io](http://purity-fb.readthedocs.io/).
+    Client for Purity//FB REST API (1.0 - 1.9), developed by [Pure Storage, Inc](http://www.purestorage.com/). Documentations can be found at [purity-fb.readthedocs.io](http://purity-fb.readthedocs.io/).
```

Comment:
should this description be updated for the client renaming the way that the others were?